frontend/src/journal/pages/Journals.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import remove from "../../assets/trash.svg";
+import useGetJournals from "../hooks/useJournal";
+import { formatDate } from "../utils/formatDate";
+
+const JournalsList: React.FC = () => {
+  const {
+    loading,
+    error,
+    data,
+    handleDeleteJournal,
+    handleSelectJournal,
+    selectedId,
+  } = useGetJournals();
+
+  if (loading) return <p>Loading journals...</p>;
+  if (error) return <p>Error loading journals: {error.message}</p>;
+
+  const journals = data?.getJournals
+    ? [...data.getJournals].sort((a, b) => {
+        const updatedAtDiff =
+          new Date(b.updatedAt).getTime() - new Date(a.updatedAt).getTime();
+        if (updatedAtDiff !== 0) return updatedAtDiff;
+
+        return (
+          new Date(b.createdAt).getTime() - new Date(a.createdAt).getTime()
+        );
+      })
+    : [];
+
+  return (
+    <div className="border-r border-[1px] w-[300px] h-[100vh] overflow-auto">
+      <div className="flex border-b justify-between p-2">
+        <p></p>
+        <button
+          className={`pr-3 ${
```

Comment:
This is a good candidate to use your `cn` utility functions. It helps you combine multiple classes depending on some conditional statement

For example, this would look this way:
```
className={cn("pr-3", !selectedId && "cursor-not-allowed opacity-50")}
```

Reply:
Done!